Migrations/LibraryContextModelSnapshot.cs

```diff
@@ -1,13 +1,13 @@
 ﻿// <auto-generated />
+using Book_Api.Data;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Infrastructure;
 using Microsoft.EntityFrameworkCore.Metadata;
 using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
-using WorkingWithDatabasesDevCareer.Data;
 
 #nullable disable
 
-namespace WorkingWithDatabasesDevCareer.Migrations
+namespace Book_Api.Migrations
```

Comment:
Don't rename namespaces when pushing an update to a master project unless you have sole responsibility for a project to avoid compile-time errors.